packages/inference/src/providers/replicate.ts

```diff
@@ -5,6 +5,9 @@ export const REPLICATE_API_BASE_URL = "https://api.replicate.com";
 type ReplicateId = string;
 
 export const REPLICATE_SUPPORTED_MODEL_IDS: ProviderMapping<ReplicateId> = {
+	conversational: {
```

Comment:
I chose `conversational` here as the task type based on the existing Together AI code.